lib/server/chat-messages.service.ts

```
@@ -11,7 +11,7 @@ export function createChatMessagesService() {
 }
 
 class ChatMessagesService {
-  constructor() {}
+  constructor() { }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)